docs/src/views/FAQ.tsx

```diff
@@ -111,6 +111,26 @@ export default () => (
       }
     />
 
+    <Question content="What's the difference between Flex, Grid, Layout, Box and Segment components?" />
+    <Answer
+      content={
+        <p>
+          Flex, Grid and Layout components handle layout aspects. The Flex component is for laying
+          out items in one direction, while the Grid component is made for two dimensional layouts.
+          Visit the {link('Layout guide', '/layout')} page for a detailed comparison between Flex
+          and Grid. The Layout component is now deprecated. Its purpose was arrangement of the
+          content of a component. Flex or Grid component should be used instead of the Layout
+          component.
+          <br />
+          Box is an abstract component that by default renders a `div` without any styles. Box
+          should be used carefully, in most cases {link("you don't need it ", '/components/box')}.
```

Comment:
I think we need to improve on this one:

`Box should be used carefully, in most cases you don't need it.`

We also need to tell users when they might need it. Not sure how to formulate this. Can you guys suggest something?

@layershifter @kuzhelov @mnajdova @miroslavstastny

Reply:
> `Box` is an abstract component that by default renders a `div` by default and can process defined styles. To keep your styles inside theme consider to use `createComponent()` instead of `Box`.

What about this?

Reply:
maybe we could even simplify it more, by saying that

> Box is a utility component that is often used by Stardust to implement higher-level components. By default, it renders styled `div` element.  

> The cases when client might want to use it are very exceptional, and all are about applying custom styles to rendered `div` element. However, more robust approach that won't break theming consistency is to instead create a custom component and define related styles as part of theme (link to Integrate Custom Components section).